src/result_gen_with_api.py

```diff
@@ -107,13 +107,36 @@ def get_unprocessed_scans(self):
 
         return self.api.get_scan(self.scan_metadata_path)
 
+    def get_unprocessed_scans_for_scan_version_workflow_id(self, scan_version, workflow_id, scan_metadata_path):
+        """
+        Gets unprocessed_scans from api and returns the no of scans
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+        """
```

Comment:
should we add argument description and return type?

Reply:
Added Argument Description